pkg/types/types.go

```diff
@@ -8,8 +8,9 @@ import (
 type ContainerInfo struct {
 	Name            string            `json:"name"`
 	Pid             int               `json:"pid"`
-	Image           string            `json:"image"`     // If set, _some_ name of the image imageID; it may have NO RELATIONSHIP to the users’ requested image name.
-	ImageRef        string            `json:"image_ref"` // In the format of StorageImageID.StringForOutOfProcessConsumptionOnly(), or "".
+	Image           string            `json:"image"`         // If set, _some_ name of the image imageID; it may have NO RELATIONSHIP to the users’ requested image name.
+	ImageRef        string            `json:"image_ref"`     // In the format of StorageImageID.StringForOutOfProcessConsumptionOnly(), or "".
+	ImageDigests    []string          `json:"image_digests"` // Possible repo digests of the used container image.
```

Comment:
Is this exactly what the users need?

- If the container is created after pulling image, and we conceptually know the _right_ repo@digest combination (at the time of pull, and we return a repo@digest value from pull), shouldn’t this ideally contain only that one? The documentation of this field seems to commit us to returning all of RepoDigests (“of the used container image”, not “possibly used when creating this container”).
- If the container is created after ImageStatus (and we don’t know the _right_ repo@digest right now, what are the users going to do? Combine the container’s image repo with this new set of RepoDigests to find out the right digest? That could work (but it is imprecise in case the precise digest matters, e.g. WRT presence of signatures) _except_ that this struct does not give them the repo of the image. As documented above, `Image` is `ImageResult.SomeNameOfThisImage`, and has no relationship with the user-requested repo.
- What is the long-term plan? Is this API eventually going to return the one right value? Or is this going to be completely deprecated in favor of container statuses in Pod.Status? If the latter, that should probably be already documented here right now.

(Starting at the exposed API; implementation details would follow from these decisions.)